oasislmf/pytools/common/event_stream.py

```diff
@@ -254,7 +254,7 @@ def register_streams_in(selector_class, streams_in):
         """
         main_selector = selector_class()
         stream_data = []
-        for stream_in in streams_in:
+        for file_id, stream_in in enumerate(streams_in):
```

Comment:
I generally have tried to differentiate id from index,
I think we should call this one file_i or file_idx